snow/engine/snowman/block/vm.go

```diff
@@ -13,10 +13,10 @@ import (
 
 // ChainVM defines the required functionality of a Snowman VM.
 //
-// A Snowman VM is responsible for defining the representation of state,
-// the representation of operations on that state, the application of operations
-// on that state, and the creation of the operations. Consensus will decide on
-// if the operation is executed and the order operations are executed in.
+// A Snowman VM defines the representation of state, operations on that state,
+// application of the operations on that state, and the creation of the operations.
+// The consensus decides whether to execute the operation, and the order that
```

Comment:
I feel like `consensus` reads as an adjective, but is used as a noun... Which reads weirdly to me.

I think I'm more okay with `Consensus decides`, rather than `The consensus decides`, because that forces the interpretation as a noun... (not an English major lol)... I'd also be happy with `The consensus engine decides`.

Maybe @danlaine would have more structured feedback here... (and maybe my brain just isn't working right now)